lib/engine/part/base.rb

```diff
@@ -9,6 +9,12 @@ def id
         "#{tile.id}-#{index}"
       end
 
+      def is?(other)
```

Comment:
why do you need this can't you just do part == part?

Reply:
Ran into an issue that I think stemmed from `Engine::Part::City#==` where the different cities on an OO tile evaluated as equal to each other and I needed to differentiate them. May have made more sense to add the `id` (and possibly `tile.id`) check to `==` instead of creating a new method.

Anyway your merge of the new routing stuff you got rid of that `==` overload and everything seems to be working fine with c4c6302